mbtools/inject_ib_ids.py

```diff
@@ -0,0 +1,42 @@
+import argparse
+from pathlib import Path
+from mbtools.models import MoodleHtmlElement
+from lxml import etree
+
+CLASSES_TO_INJECT = [
+    "os-raise-ib-input",
+    "os-raise-ib-pset",
+    "os-raise-ib-pset-problem"]
+
+
+def inject_ib_ids(html_dir):
+    all_files = []
+    for path in Path(html_dir).rglob('*.html'):
+        all_files.append(path)
+
+    for file_path in all_files:
+        with open(file_path, 'r') as f:
+            parent_string = '<content></content>'
+            parent_element = etree.fromstring(parent_string)
+            parent_element.text = f.read()
+            content = MoodleHtmlElement(parent_element, str(file_path))
+
+        content.add_ib_content_id(CLASSES_TO_INJECT)
+
+        with open(file_path, 'w') as f:
+            f.write(content.tostring())
```

Comment:
In so much as I think you'll want to write the file out using the same incantation as [here](https://github.com/openstax/raise-mbtools/blob/e8a75dffb94ce66f5185fadb901b10ff3e362c46/mbtools/fix_html.py#L21), we might want to create a shared utility function like we did [here](https://github.com/openstax/raise-mbtools/blob/e8a75dffb94ce66f5185fadb901b10ff3e362c46/mbtools/utils.py#L6-L10) (maybe `write_html_soup` or something :man_shrugging: ) that both scripts use.